logins.js

```diff
@@ -25,10 +25,16 @@ module.exports = {
                 return callback(err, null);
             }
             
+            if (body.trim().indexOf('<') === 0) {
```

Comment:
Is there any possibility of a `<` being in the body of a valid request, such as in a string? I haven't looked at request bodies at all so I'm just proposing a possibility and I'm not sure if this will be an issue.